apps/channels/models.py

```diff
@@ -12,16 +12,42 @@
 from apps.web.meta import absolute_url
 
 
+class ChannelPlatform(models.TextChoices):
+    TELEGRAM = "telegram", "Telegram"
+    WEB = "web", "Web"
+    WHATSAPP = "whatsapp", "WhatsApp"
+
+    @classmethod
+    def for_dropdown(cls):
+        return [
+            cls.TELEGRAM,
+            cls.WHATSAPP,
+        ]
+
+    def form(self):
+        from apps.channels.forms import ChannelForm
+
+        return ChannelForm(initial={"platform": self})
+
+    def extra_form(self, *args, **kwargs):
+        from apps.channels.forms import TelegramChannelForm, WhatsappChannelForm
```

Comment:
Can this be moved to the top of the file, or were there circular import errors?

Reply:
It can't because forms imports from models